depth.py

```diff
@@ -8,13 +8,16 @@ def depth(filename):
 
     for path in g.paths:
         for segment in path.segment_names:
-            name = segment.name
+            name = int(segment.name)
```

Comment:
Do we know that segment names are always numbers? (If not, then this call to `int(...)` might crash.)